R/remove_phage.R

```diff
@@ -4,18 +4,26 @@
 #'
 #' @param virion_data Data frame. Virion data to be processed
 #' @param phage_taxa Data frame. Phage taxa from vmr data.
+#' @param taxa_ranks Character. Vector of taxa ranks. See ICTV taxonomy for ranks.
 #'
 #' @returns
 #' @export
 #'
 #' @examples
-remove_phage <- function(virion_data = virion_unprocessed,phage_taxa){
+remove_phage <- function(virion_data = virion_unprocessed,
+                         phage_taxa, 
+                         taxa_ranks = c("order","family","genus","species")){
```

Comment:
makes it easier to control the taxanomic ranks used in the filter. Class was causing a problem because of mismatches between ictv and ncbi